web/src/click_handlers.js

```diff
@@ -573,6 +573,28 @@ export function initialize() {
             get_target_node,
             check_reference_removed,
         );
+
+        /*
+           We need a separate way to handle mouseevents for status-emoji in sidebars apart from tippyjs.ts logic. This is because status-emoji is a part of buddy-list tooltip trigger and we need to separate both the tooltips to appear at the same time. Hence when we enter the emoji section, the exact buddy-list tooltip instance needs to be destroyed. Also  when we leave the emoji section , the buddy list tooltip needs to be re-rendered.
+        */
```

Comment:
Please line-wrap your comments (as we do everywhere else in the code base) and make them more concise.

Reply:
Hope the present changes works.